src/dsync/dsync.c

```diff
@@ -63,6 +63,7 @@ static void print_usage(void)
     printf("  -b  --batch-files <N>   - batch files into groups of N during copy\n");
     printf("      --bufsize <SIZE>    - IO buffer size in bytes (default " MFU_BUFFER_SIZE_STR ")\n");
     printf("      --chunksize <SIZE>  - minimum work size per task in bytes (default " MFU_CHUNK_SIZE_STR ")\n");
+    printf("  -e, --xattrs <OPT>      - copy xattrs (none, all, non-lustre, libattr)\n");
```

Comment:
We've got ``e`` instead of ``X`` here, too

Reply:
oops, fixed.